seven_half/game.py

```diff
@@ -1,3 +1,5 @@
+from typing import List, Any
+
```

Comment:
- `Any` is redundant import.
- `List` is redundant, use built-in `list` to type hint.